### PR TITLE
example: add pyth oracle anchor interface

### PR DESCRIPTION
**Pull request by NorbertBodziony**

### CODE REVIEW

File: examples/pyth/programs/pyth/src/pc.rs [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+}
+impl Default for PriceType {
+    fn default() -> Self {
+        PriceType::Price
+    }
+}
+#[derive(Default, Copy, Clone)]
+#[repr(C)]
+pub struct Price {
+    pub magic: u32,       // pyth magic number
```
Comment (armaniferrante) [inline examples/pyth/programs/pyth/src/pc.rs]: Please start and end all comments with `Uppercase and a period.`

**Review by armaniferrante — COMMENTED** [review]
Handled some nits, but posting for future reference.

File: examples/pyth/programs/pyth/src/pc.rs (hunk 1 above)
Comment (NorbertBodziony) [reply examples/pyth/programs/pyth/src/pc.rs]: Thanks will do.

**Review by armaniferrante — APPROVED** [review]
LGTM once the tests pass.

**Comment by armaniferrante** [comment]
Tests failed due a travis problem (unrelated to this change). Rerunning.

**Pull request merged**